### PR TITLE
Chore/use local messaging

### PR DESCRIPTION
**Pull request by santiagonoguez**
This is the code that tests locally the full HTML page with image data component and RLS, and is now working on player electron.

### CODE REVIEW

File: src/image-remote-transpiled.html [hunk 1]
```diff
@@ -27,17 +40,27 @@
...
 
-        document.querySelector('#rise-data-image-01').addEventListener('url-updated', event => {
+        // all of the following will be implemented later by a rise-image component
+        let image01 = document.querySelector('#rise-data-image-01');
+        image01.addEventListener('image-url-updated', event => {
+          document.querySelector('#image01').innerHTML = `
+            <img src=${ event.detail.url } width="50" height="50">
+          `;
+        });
+        image01.addEventListener('image-not-available', event => {
```
Comment (stulees) [inline src/image-remote-transpiled.html]: @santiagonoguez As mentioned in [rise-data-image PR](https://github.com/Rise-Vision/rise-data-image/pull/6) I don't think this appropriate for all FILE-UPDATE _status_ other than "CURRENT".

Comment (santiagonoguez) [reply src/image-remote-transpiled.html]: @stulees I'll make the change and fuse these two events into one !

File: src/image-remote-transpiled.html [hunk 2]
```diff
@@ -14,10 +14,23 @@
...
     </script>
+    <script src="http://widgets.risevision.com/staging/common/rise-player-configuration.js"></script>
+    <script src="http://widgets.risevision.com/staging/common/rise-local-messaging.js"></script>
+    <script src="http://widgets.risevision.com/staging/common/rise-local-storage.js"></script>
+    <script>
+      console.log("configuring");
+
+      // electron / websocket or chromeos / window
+      RisePlayerConfiguration.configure({}, {
+        player: "electron",
```
[the remark this answers is not in the document]
Comment (santiagonoguez) [reply src/image-remote-transpiled.html]: @stulees my bad ! but during my tests yesterday I was doing this change whenever I changed player, so I'm pretty sure local environment has issues. But I'll test again all scenarios, though !

**Comment by santiagonoguez** [comment]
@stulees I updated the page with the latest event handling, please review again !

**Pull request merged**